.github/workflows/Linux.yml

```diff
@@ -142,7 +142,7 @@ jobs:
           git fetch origin gh-pages
           git checkout --force gh-pages
           rsync --recursive build/doc/ .
-          git add animview/ corsixth_engine/ corsixth_lua/ index.html leveledit/
+          git add animview/ check-for-updates corsixth_engine/ corsixth_lua/ index.html leveledit/
```

Comment:
Is check-for-updates here?

Reply:
It is now: http://corsixth.github.io/CorsixTH/check-for-updates